Cargo.toml

```diff
@@ -23,6 +27,7 @@ authors = [
   "Ben Bangert <ben@groovie.org>",
   "Phil Jenvey <pjenvey@underboss.org>",
   "Mozilla Services Engineering <services-engineering+code@mozilla.com>",
+  "Eragon <eragon+github@eragon.re>",
```

Comment:
:heart: (Seriously, yes, you should add your name here. You're doing a lot.)

Reply:
Thank you